app/assets/javascripts/datepicker_head.js

```diff
@@ -0,0 +1,3 @@
+//= require jquery
+//= require jquery_ujs
+//= require jquery.cookie
```

Comment:
Oh, one 1. Why do we need jquery.cookie?

Reply:
hmmm I suppose we don't. I'll remove.